webapp/src/AppComponents/UploadFileDialog.tsx

```diff
@@ -113,30 +113,10 @@ function UploadFileDialog(properties: {
         <Cross
           style={{ marginRight: 12 }}
           onClick={handleClose}
-          ref={crossRef}
```

Comment:
This is an error you *need* ref in this case, because in this case something needs to have the focus and there is no input in the dialog.

We could also have the focus on the UploadTitle, but something needs to have the focus

Reply:
Added back, sorry!